swan-lake/development-tutorials/source-code-dependencies/organize-ballerina-code.md

```diff
@@ -12,9 +12,9 @@ intro: The sections below include information about packages and how you can man
 
 Writing code in an organized manner from the beginning of the project is important for the lifecycle of the project and its maintainability in the long run. Organized code will make it easy to extend and improve your project over time. Ballerina project structure makes it easy to write clean code by eliminating repetitions, writing reusable code, adding new features without changing the existing code, etc. To achieve this, Ballerina has the concept of packages and modules. 
 
-Ballerina code is organized in a single shareable unit called a `package`.
+Ballerina code is organized in a single shareable unit called a package.
 
-A `package` is a collection of `modules`, and a `module` is a collection of Ballerina source files, test files, and resources. A package should contain at least one module called the default module. Each module has its own directory, which organizes source files, test files, and resources.
+A package is a collection of modules, and a module is a collection of Ballerina source files and test files. A package should contain at least one module, which becomes the default module. Each module has its own directory, which organizes source files and test files. A package can contain resources and test-only resources specific to the package in the package source root.
```

Comment:
How is a resource defined as test-only?

Reply:
when the resources are added to a `resources` directory within `tests` directory.

Reply:
```suggestion
A package is a collection of modules, and a module is a collection of Ballerina source files and test files. A package should contain at least one module, which becomes the default module. Each module has its own directory, which organizes source files and test files. A package can include resources in a `resources` directory located at the package source root, and test-specific resources in a `resources` directory within the `tests` directory at the package source root.
```

Reply:
If we have test-specific resources for submodules, they also have to go in the tests directory in the root?

Reply:
we don't support resources for sub modules. Resources are applicable to the entire package not module specific.